tests/framework/MCMC/test_10D_proposal_mvn.xml

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="silent">
```

Comment:
should we create a new Requirement "satisfied" by these implementations? In my opinion we should.
Can you propose one?

Reply:
Added